en/tools/config-catalog-generator-cc/data/router/configs.json

```diff
@@ -349,6 +349,34 @@
       ],
       "exampleFile": "upstream_tls.toml"
     },
+    {
+      "title": "Downstream TLS",
+      "options": [
+        {
+          "name": "router.downstream.tls",
+          "required": false,
+          "description": "The configurations for SSL configuration related to the downstream in Choreo Connect.",
```

Comment:
The configurations for SSL related to the downstream in Choreo Connect.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/7065/commits/4ac3b72b3278ab8265e5f4635499d5ea37db0c04